src/main/java/com/redhat/devtools/lsp4ij/LanguageServiceAccessor.java

```diff
@@ -106,14 +106,18 @@ private void checkCurrentlyOpenFiles(Collection<LanguageServerDefinition> defini
         for (LanguageServerDefinition definition : definitions) {
             Project[] projects = ProjectManager.getInstance().getOpenProjects();
             for (Project p : projects) {
-                VirtualFile[] files = FileEditorManager.getInstance(p).getOpenFiles();
-                for (VirtualFile file : files) {
-                    findAndStartLsForFile(file, definition);
-                }
+                findAndStartLanguageServerIfNeeded(definition, p);

```

Comment:
maybe catch exceptions so a failure won't prevent other definitions/projects to be handled

Reply:
fixed